makefile

```diff
@@ -1,4 +1,4 @@
-flist = 2 3 4 5 5e 6 EV1 EV3 EV4
```

Comment:
The log-transform results in a more linear relationship in the EV4 plots. We also don't need 5e since all the receptors are on a similar scale.